server/home.go

```diff
@@ -17,7 +17,7 @@ func (s *Server) handleHomePage(w http.ResponseWriter, r *http.Request) {
 			username = user.Username
 		}
 	}
-	tmpl, _ := template.ParseFiles("views/index.html")
```

Comment:
Please just name the folder `views` as it was, there's no reason to change the name at all.